lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1909,6 +1909,12 @@ def each_host_range_chunk(host_ranges, &block)
   #######
 
   def add_data_service(*args)
+    # database is required to use Mdm objects
+    unless framework.db.active
+      print_error("Database not connected; connect to an existing database with db_connect before using data_services")
```

Comment:
I feel like longer term we should move everything under a single command (either db_connect or data_services). Agree/disagree?

Reply:
Agree